src/Command/I18nExtractCommand.php

```diff
@@ -692,6 +693,9 @@ protected function checkUnchanged(string $oldFile, int $headerLength, string $ne
             return false;
         }
         $oldFileContent = file_get_contents($oldFile);
+        if ($oldFileContent === false) {
+            throw new CakeException(sprintf('Cannot read file content of `%s`', $oldFile));
```

Comment:
I don't think this is an error case. Looking at the `_writeFiles()` which calls this method it gets called for all files to be written may or may not already exist.

Reply:
Well, it should definitly not happen
Right before this we have
```
if (!file_exists($oldFile)) {
            return false;
        }
```
So it is pretty much an exception case if it then cannot be read.

Reply:
Ah yes, I misread the code. So all good here.